static/app/data/forms/projectGeneralSettings.tsx

```diff
@@ -64,8 +64,10 @@ export const fields: Record<string, Field> = {
     },
 
     saveOnBlur: false,
-    saveMessageAlertType: 'info',
-    saveMessage: t('You will be redirected to the new project slug after saving'),
+    saveMessageAlertType: 'warning',
+    saveMessage: t(
+      'Changing project name will change your project slug too and can break your build scripts. Are you sure you want to change name?'
```

Comment:
this copy reads a little weird to me. how about something like

```suggestion
      'Changing a project's name will also change the project slug. This can break your build scripts! Please proceed carefully.'
```